### DISCUSSION TITLE
Pycharm Error by sharing project on GitHub

### DISCUSSION THREAD

**Discussion by timo2009** [community/community#144063] [New to GitHub]
Hello everyone

I have a problem with Pycharm:
I have developed a project there and would now like to share it on github using the share project on github function.
Problem: I go to the corresponding window, select my account and click on “share”, I get the error: “Cannot load informations for github.com/username:Request response: Access to this site has been restricted. If you belive this is an error, pleas contact support.github.com.

Please help me.
Timo

Translated with DeepL.com (free version)

**Comment by Srishti44-g** [comment]
Hi Timo!

It sounds like you're experiencing a permissions or connectivity issue between PyCharm and GitHub. Here are some steps you can try to resolve it:

1. Check Your GitHub Access: Make sure you can access GitHub via your browser without restrictions. If you’re on a network with restricted access (like school or workplace Wi-Fi), try switching to a different network.

2. Re-authenticate in PyCharm:

Go to File > Settings > Version Control > GitHub (or Preferences on Mac).

Remove the existing GitHub account, then add it back by re-authenticating with your GitHub credentials.
You may want to use Token Authentication instead of username/password, as GitHub no longer supports password-based authentication. Generate a Personal Access Token on GitHub [here](https://github.com/settings/tokens) and paste it in PyCharm.

3. Check Proxy Settings:

If you're behind a proxy, make sure PyCharm is configured to connect through it. Go to File > Settings > Appearance & Behavior > System Settings > HTTP Proxy and configure your proxy settings accordingly.

4. Test with Command Line Git:

Open a terminal in PyCharm or your system terminal and try pushing your project to GitHub using Git commands:

git remote add origin https://github.com/username/repository.git
git push -u origin master

If this works, it may be a PyCharm-specific issue, in which case reinstalling or updating PyCharm might help.

5. Contact GitHub Support:

If the issue persists, reach out to GitHub Support directly at
https://support.github.com/
Describe the error message and any steps you've already taken.
Let me know if these steps help!

**Reply by timo2009** [reply]
Hi,
Thank you, I was able to connect the project to Git via the terminal and from then on, I was able to push and commit with Pycharm again.
So it is simply a problem with Pycharm.
Thank you very much!
Timo

**Reply by ghostinhershell** [reply]
Hi @timo2009 👋🏾,It looks like @Srishti44-g's response was the answer to your question here. Can you please mark their response as the answer? When you mark a question as an answer, GitHub will highlight the comment and replies to the comment to help visitors quickly find the answer.
![image](https://github.com/user-attachments/assets/b89e7161-c2d1-47e2-a058-242df633e6b2)
